pkg/apis/triggers/v1alpha1/event_listener_types.go

```diff
@@ -26,12 +27,25 @@ import (
 var _ apis.Validatable = (*EventListener)(nil)
 
 type EventListenerSpec struct {
-	TriggerBindingRefs []TriggerBindingRef `json:"triggerbindingrefs,omitempty"`
+	Triggers []Trigger `json:"triggers,omitempty"`
+}
+
+// +k8s:deepcopy-gen=true
+type Trigger struct {
+	TriggerBinding     TriggerBindingRef  `json:"triggerBinding"`
+	TriggerTemplate    TriggerTemplateRef `json:"triggerTemplate"`
+	ServiceAccountName string             `json:"serviceAccountName"`
 }
 
 type TriggerBindingRef struct {
-	Name       string `json:"name,omitempty"`
-	Namespace  string `json:"namespace,omitempty"`
+	Name       string             `json:"name"`
+	APIVersion string             `json:"apiversion,omitempty"`
+	Params     []pipelinev1.Param `json:"params,omitempty"`
```

Comment:
hm do `params` need to be here? what does it mean to pass params to `triggerBinding`? (might be something that can be cleared up by comments + examples)

Reply:
These were removed